packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -233,10 +248,12 @@ export class GCFBootstrapper {
             // TODO: add unit tests for both forms of payload.
             payload = this.parsePubSubPayload(request);
           }
+          // TODO: find out the best way to get this type, and whether we can
+          // keep using a custom event name.
           await this.probot.receive({
-            name,
+            name: name as EventNames.StringNames,
             id,
-            payload: payload,
+            payload: await this.maybeDownloadOriginalBody(payload),
```

Comment:
From a readability perspective, I usually shy away from inline `await`s like this.  Would assignment outside of the receive call be ok?  Also a comment on why this is a "maaaaaaybe" would be nice 😆

Reply:
done.